compiler-core/src/fix/bit_string_rename.rs

```diff
@@ -50,7 +50,11 @@ impl Fixer {
 
             Definition::Import(i) => {
                 if i.module == "gleam" {
-                    self.prelude_module_import_alias = Some(i.used_name());
+                    self.prelude_module_import_alias = match i.clone().as_name {
```

Comment:
We don't need to clone the entire import here. Generally cloning should be avoided, and when it is done it should only be with the smallest item required, so it is almost always on the right hand side of the `=>`. Here we can take this by reference and then clone the name if needed.

```suggestion
                    self.prelude_module_import_alias = match i.as_name.as_ref() {
```